controller/wingsail/wingsail_ctrl_node.py

```diff
@@ -78,9 +80,29 @@ def __init_publishers(self):
         """Initializes the publishers of this node. Publishers update ROS topics so that other ROS
         nodes in the system can utilize the data produced by this node.
         """
-        # TODO Implement this function by initializing publishers for topics that give the desired
-        # output data
-        pass
+
+        self.get_logger().debug("Initializing publishers...")
+        self.__trim_tab_angle_pub = self.create_publisher(
+            msg_type=SailCmd,
+            topic="sail_cmd",
+            qos_profile=1,
+        )
```

Comment:
For the topic name string, it's best to store it as a constant and access it rather than just using a magic string. However, there is a particular way that it's done in the boat simulator that should be consistent here as well.

Can you just add a `TODO` comment for now to make a note of this, but for now we can leave it be. I will be making a separate issue for this.